cpp/src/arrow/filesystem/gcsfs.h

```diff
@@ -230,9 +230,7 @@ class ARROW_EXPORT GcsFileSystem : public FileSystem {
       const std::string& path,
       const std::shared_ptr<const KeyValueMetadata>& metadata) override;
 
-  /// Create a GcsFileSystem instance from the given options.
```

Comment:
We don't need to remove this comment too. Could you revert this change?

Reply:
Reverted! Ref e5b606e462cb362ef49092eb10797d166e2e60fd.